src/bin/cli.js

```diff
@@ -0,0 +1,131 @@
+import Preferences from 'preferences';
+import chalk from 'chalk';
+import createLogger from 'cli-logger';
+import md5 from 'md5';
+import request from 'request';
+import url from 'url';
+
+export const SERVER = process.env.CRYPTID_SERVER || 'https://cryptid.adorable.io';
+
+function buildUrl(path) {
+  return url.resolve(SERVER, path);
+}
+
+function loadSettings() {
+  let name = `com.adorable.cryptid.${md5(SERVER)}`;
+  let settings = new Preferences(name, {server: SERVER, token: '', email: ''});
+
+  settings.isLoggedIn = settings.token.length > 0 && settings.email.length > 0;
```

Comment:
is it ever possible that settings could end up undefined or null?

Reply:
No, if it doesn't exist then it uses the template on line 16 to create a new instance.